virt/virtual_machines/advanced_vm_management/virt-configuring-vgpu-passthrough.adoc

```diff
@@ -0,0 +1,25 @@
+[id="virt-configuring-vgpu-passthrough"]
+= Configuring vGPU passthrough for virtual machines
+include::modules/virt-document-attributes.adoc[]
+:context: virt-configuring-vgpu-passthrough
+
+toc::[]
+
+You can configure a virtual GPU (vGPU) passthrough for PCI host devices or mediated hardware devices for access by your cluster and virtual machines. Configuring a vGPU passthrough allows you to:
```

Comment:
suggestion: `You can configure a virtual GPU (vGPU)` ==> `Configure a virtual Graphics Processing Unit (vGPU)`

Reply:
Just a suggestion for the first sentence, as it's a bit complexed (line 8): "Your cluster and virtual machines can access PCI host devices or mediated hardware devices through a vGPU passthrough configuration". WDYT?

Reply:
Thanks this is much better. I do prefer to lead with verbs like Configure. Sometimes I push back in peer review from writers who use "you" a lot. "You" is OK, but I think we overuse it.